streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamsPartitionAssignor.java

```diff
@@ -169,28 +144,21 @@ void addConsumer(final String consumerMemberId,
         @Override
         public String toString() {
             return "ClientMetadata{" +
-                    "hostInfo=" + hostInfo +
-                    ", consumers=" + consumers +
-                    ", state=" + state +
-                    '}';
+                "hostInfo=" + hostInfo +
+                ", consumers=" + consumers +
+                ", state=" + state +
+                '}';
         }
     }
 
-    private static final class InternalStreamsConfig extends StreamsConfig {
-        private InternalStreamsConfig(final Map<?, ?> props) {
+    private static final class QuietStreamsConfig extends StreamsConfig {
```

Comment:
just making this a little more readable.

Reply:
Unfortunately, that's in test-utils.

I could move it to a separate internal class in this module, and then use it in test-utils, though.

Reply:
Yeah, let's do that: we should try our best to avoid code duplicates.